src/wallet/init.cpp

```diff
@@ -16,8 +16,8 @@
 std::string GetWalletHelpString(bool showDebug)
 {
     std::string strUsage = HelpMessageGroup(_("Wallet options:"));
-    strUsage += HelpMessageOpt("-addresstype", strprintf(_("What type of addresses to use (\"legacy\", \"p2sh-segwit\", or \"bech32\", default: \"%s\")"), FormatOutputType(OUTPUT_TYPE_DEFAULT)));
```

Comment:
I would rather improve the description.

BTW, currently RPC help is very clear:

https://github.com/bitcoin/bitcoin/blob/898f560b55aba2070f188b0223ef2beefcdede8b/src/wallet/rpcwallet.cpp#L147